week4/48_rotate_image.py

```diff
@@ -0,0 +1,21 @@
+# https://leetcode.com/problems/rotate-image/
```

Comment:
I am bit embarrassed to show my code 😅, yours is nice and simple. I think you also took less time than me to solve this. I quite struggled with this problem. I'd love to hear your explanation as well.